query-node/mappings/src/content/utils.ts

```diff
@@ -0,0 +1,382 @@
+// TODO: check all `db.get()` and similar calls recieve a proper type argument (aka add `.toString()`, etc. to those calls)
+// TODO: can we rely on db having "foreign keys"? When item is deleted will automaticly be all relations to it unset?
+//       Similarly, will saving item also save all its related items no-yet-saved in db, or do they need to saved individually?
+//       Also, is the assumption that `db.save(MyType, {myProperty: undefined})` unsets value in db correct?
```

Comment:
Can we rely on these assumptions, or we need to handle db save/delete for related items manually?

Reply:
The first TODO was resolved. The second one can be rearticulated into:
```
// assuming there is 1:N relation between MySecondaryItem and MyMainItem
const mySecondaryItem = new MySecondaryItem({someData})
const myMainItem = new MyMainItem({mySecondaryItem})

await db.save<MyMainItem>(myMainItem)
// is `mySecondaryItem` also saved in db now?
await db.remove<MyMainItem>(myMainItem)
// is `mySecondaryItem` also removed from db now? 
```